js/gem-notice.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
The dismiss function works out of the box, but it only hides the container. The option will still exist in the database causing the notice to display again.

This file fires an AJAX function that removes the option from the database, preventing the notice from displaying on subsequent page loads.